packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStart/ForceFunctionContainerlessStartExecutor.ts

```diff
@@ -51,10 +56,8 @@ export class ForceFunctionContainerlessStartExecutor extends ForceFunctionStartE
 
     localRun
```

Comment:
nit: should we await this now that it resolves with the info we need to proceed?

Reply:
Good point.- but changing it to to use await requires making the `startFunction` as async which makes it different from `ForceFunctionContainerStartExecutor` and that could lead to inconsistencies when calling them from `ForceFunctionStartExecutor` ?

Reply:
hum...true.  I think it's the lessor evil to make the startFunction return a `Promise<void>` verse having the unhandled promise.  Then we can have the async in the containerless version and the container version can just return a resolved promise.

Reply:
I'm curious if this will fix the problem where VSCode says the containerless function has started, but then if you try to invoke too quickly it's not actually started and the invoke fails.

Reply:
👍 Refactored both container and containerless start to be async.